benchmarks/mmmu/README.md

```diff
@@ -0,0 +1,25 @@
+# MMMU: A Massive Multi-discipline Multimodal Understanding and Reasoning Benchmark for Expert AGI
+
+[MMMU](https://arxiv.org/abs/2311.16502) is a benchmark for evaluating multi-modal models on college-level tasks across a variety of subjects. 11.5k problems are gathered across 30 subjects and image types. Around 93% of questions in the evaluation dataset are multiple-choice, with the remainder being open-ended. GPT-4o is reported to achieve 63-69% accuracy for MMMU.
+
+## Execution
+Here is an example from the dataset:
+```
+Question: The double-tiered columns allowed for all of the following EXCEPT (<image 1>)
+Option:
+A) barrel-vaulted roofing
+B) decorative rhythm and repetition
+C) a higher roof to make up for the short columns
+D) the entrance of light and air into the hall
+```
+![alt text](image.png)
```

Comment:
Noted that this image is 1.25MB. Since `image.png` is just for the README could we either:

1) Link to a version of it somewhere else online; or
2) Create a subtantially compressed version of it (e.g. lower quality jpeg)

Reply:
Opted for (2), pushed!